stock-tracker/src/features/overview/redux/actions.ts

```diff
@@ -0,0 +1,45 @@
+import {
+  ADD_COMPANY_OVERVIEW,
+  SET_LOADING_OVERVIEW,
+  SET_ERROR_OVERVIEW
+} from "./actionTypes";
+import { ActionWithPayload, Action } from "../../../utils/actions";
+import * as constants from "./actionTypes";
```

Comment:
nit: I think it's better to import the constants you need, rather than getting them all `*`

Reply:
Have just changed and committed this.